src/vivarium_census_prl_synth_pop/constants/metadata.py

```diff
@@ -126,8 +127,11 @@
     8: "Multiracial or Other",
     9: "Multiracial or Other",
 }
+RACE_ETHNICITIES = set(RACE_ETHNICITY_VAR_MAP.values())
```

Comment:
Why is this one a set while the others are lists? Doesn't this create an issue with depending on the set iteration order?

Reply:
It's a set because I needed to remove duplicates, and as I said above, we don't actually care about the order. That being said, consistency is good, and it's also better to know the order in case do care later